src/_pytest/terminal.py

```diff
@@ -463,6 +469,46 @@ def write_sep(
         self.ensure_newline()
         self._tw.sep(sep, title, fullwidth, **markup)
 
+    def write_hrule_single(
+        self,
+        title: Optional[str] = None,
+        fullwidth: Optional[int] = None,
+        **markup: bool
+    ) -> None:
+        self.write_sep(
+            "\u2500" if self.config.option.unicode else "-", title, fullwidth, **markup
```

Comment:
The escaping is not needed here, I suggest using the character directly (same for the others):

```suggestion
            '─' if self.config.option.unicode else "-", title, fullwidth, **markup
```

Reply:
But when we're using the character directly in the file, we have to make sure, that the file uses the right encoding (i.e. UTF-8). But it seems that we've already got unicode characters in the pytest sourcecode:
```python
>>> import pathlib
>>> max(max(pys.read_text(encoding="utf-8")) for pys in pathlib.Path(".").glob("*/*.py"))
'😊'
```

However, I find `─` to be visible nearly indistinguishable from `-` (at least in non-proportional fonts), so the source code might be misleading. Using the escaped character clearly shows whats going on.

Reply:
And of course be could always use an `\N` escape, i.e.:
```python
"\N{BOX DRAWINGS LIGHT HORIZONTAL}"
```

Reply:
The file encoding is UTF-8 so it's not a problem.

> However, I find ─ to be visible nearly indistinguishable from -

I'd like to have the symbol because then I can see what it is. Maybe leave the symbol, but add a clarifying comment with its Unicode name?